gossip/emitter/emitter.go

```diff
@@ -127,13 +128,11 @@ func (em *Emitter) init() {
 
 // Start starts event emission.
 func (em *Emitter) Start() {
-	if em.config.Validator.ID == 0 {
+	if em.config.Validator.ID == 0 || em.done != nil  {
```

Comment:
Will need to update the comment to "short circuit if not a validator or already started"